FirebaseAuth/Sources/Swift/Auth/AuthDataResult.swift

```diff
@@ -55,9 +54,13 @@ import Foundation
     self.credential = credential
   }
 
-  public static var supportsSecureCoding: Bool {
-    return true
-  }
+  // MARK: Secure Coding
+
+  private let kAdditionalUserInfoCodingKey = "additionalUserInfo"
+  private let kUserCodingKey = "user"
+  private let kCredentialCodingKey = "credential"
+
+  public static var supportsSecureCoding = true
```

Comment:
With the public conformance removed, should this be internal?
```suggestion
  static var supportsSecureCoding = true
```

Reply:
![Screenshot 2024-01-12 at 9 52 01 AM](https://github.com/firebase/firebase-ios-sdk/assets/73870/9799337c-26cb-42a3-b624-07ce1c9a362a)